glossary.yml

```diff
@@ -1377,6 +1377,12 @@
     def: >
       Haifanyiki kwa wakati mmoja. Katika programu, operesheni ya asynchronous ni ile inayoendesha kwa kujitegemea ya nyingine, 
       au ambayo huanza wakati mmoja na kuishia kwa mwingine.
+
```

Comment:
@elletjies could you please help us and clarify about blank lines? The contributor's guide says "Ensure there is one blank line between the end of each block of translations in a slug" but shows an example of too many lines between two different slugs. As we can see above, there are mostly no blank likes within a slug, but sometimes they happen. What should we do? Is such translation acceptable, or do we need to remove the blank line?

Reply:
@elletjies thanks! so we will accept that, if there will be no other corrections, otherwise we will also remove the blank line. In the future, we will try to avoid them within slugs.